--- spin-4.1.3/Src4.1.3/make_unix.orig	2005-06-26 18:09:46.000000000 +0000
+++ spin-4.1.3/Src4.1.3/make_unix	2005-06-26 18:13:09.000000000 +0000
@@ -18,8 +18,8 @@
 #			and add $(CFLAGS) to the spin.o line: $(CC) $(CFLAGS) -c y.tab.c
 #	on __FreeBSD__: omit -D_POSIX_SOURCE
 
-YACC=yacc		# on Solaris: /usr/ccs/bin/yacc
-YFLAGS=-v -d 		# creates y.output and y.tab.h
+YACC=bison		# on Solaris: /usr/ccs/bin/yacc
+YFLAGS=-y -d 		# creates y.output and y.tab.h
 
 SPIN_OS= spin.o spinlex.o sym.o vars.o main.o ps_msc.o \
 	mesg.o flow.o sched.o run.o pangen1.o pangen2.o \
